slave-dotnet20/Dockerfile.rhel7

```diff
@@ -0,0 +1,30 @@
+FROM openshift3/jenkins-slave-base-rhel7
```

Comment:
The other slaves have `openshift/jenkins-slave-base-rhel7` as base, but I could not pull that, so changed it to `openshift3`